src/sp/it/pl/main/App.kt

```diff
@@ -109,10 +109,13 @@ fun main(args: Array<String>) {
     // Relocate temp & home under working directory
     // It is our principle to leave no trace of ever running on the system
     // User can also better see what the application is doing
-    val tmp = File("").absoluteFile.childOf("user", "tmp")
-    isValidatedDirectory(tmp)
-    System.setProperty("java.io.tmpdir", tmp.absolutePath)
-    System.setProperty("user.home", tmp.absolutePath)
+    val tmp = File("user", "tmp").absoluteFile
+    if(isValidatedDirectory(tmp)) {
+        System.setProperty("java.io.tmpdir", tmp.absolutePath)
+        System.setProperty("user.home", tmp.absolutePath)
+    } else {
+        System.err.println("Can't write to $tmp!")
```

Comment:
There is PlayerFX.xml file. Can I ask you to update it. Involves
- setting correct classPath.mainClass
- jre.opt which are jvm arguments
- relativizing paths

Reply:
The jvm args would have to be adjusted after the environment branch is merged. Can't you do a straight commit to master after that correcting it? Or do you need help with something?

Reply:
yes, but quite a bit of what you mentioned isn't actually related to the PR